cheetunit-examples/hello-world-example/src/test/resources/cheetunit.properties

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright 2020 Gepardec IT Services GmbH and the CheetUnit contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cheetunit.host.protocol=http
+cheetunit.host.name=localhost
+cheetunit.host.port=8080
+cheetunit.path=hello/rest/cheetunit-insider
```

Comment:
The path `cheetunit-insider` ist hardcoded in cheetunit-core. Therefore we could just leave the path to the rest services, like `hello/rest`. Of course we should adapt then other configuration classes, such that the hardcoded part will be added to the path at some step.

Reply:
Good point, but I think it might be confusing if just specify the middle part of a path